web/src/main/java/com/bazaarvoice/emodb/web/scanner/scanstatus/ScanStatus.java

```diff
@@ -111,9 +113,13 @@ public void setCompleteTime(Date completeTime) {
         _completeTime = completeTime;
     }
 
-    @JsonIgnore
+    public void setCompactionControlBufferTimeInMillis(long compactionControlBufferTimeInMillis) {
+        _compactionControlBufferTimeInMillis = compactionControlBufferTimeInMillis;
+    }
+
     public Date getCompactionControlTime() {
-        return _compactionControlTime;
+        // Adding compactionControlBuffer time to the start time to give us the compaction control time. (setting the time in the future takes care of the issue of there being any in-flight compactions)
+        return DateUtils.addMinutes(_startTime, (int) Duration.ofMillis(_compactionControlBufferTimeInMillis).toMinutes());
```

Comment:
Minor: What if `_compactionControlBufferTimeInMillis` isn't an even number of minutes, such as 90 seconds.  Why not just convert to milliseconds and add?